TrafficLightsDemo/app_win.tcjs

```diff
@@ -1,39 +1,40 @@
-// Windows - Visual Studio 15
+// Windows - Visual Studio 17
 let tc = TCF.define(TCF.CPP_TRANSFORM);
-tc.pocoLoc = 'D:\\github\\poco';
 tc.sources = [
-    'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_rx2SkNXDEei3BJT-OqdJZA'
+    'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_rx2SkNXDEei3BJT-OqdJZA',
 ];
-tc.topCapsule = 'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_4gz4gNXDEei3BJT-OqdJZA';
-tc.createTargetProject = true;
-tc.targetProject = '/TrafficLightsDemo_target';
+tc.prerequisites = [
+    'platform:/resource/LibHttpServer/httpServerLib_static.tcjs',
+];
+tc.apocoLoc = '$(RSA_RT_HOME)/tcpserver/poco/';
```

Comment:
Where is this variable used? I don't see any usage of it. Also, why does it only exist for the Win TC?

Reply:
Yes, I checked it is not used. I was doing some experiment and it left like this. I will remove it.